### DISCUSSION TITLE
Welcome to IC SDK Discussions!

### DISCUSSION THREAD

**Discussion by AnthonyVanVranken** [dfinity/sdk#2115] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community and discuss improvements to the SDK. We hope that you:
  * Upvote features/topics that you think should get more attention.
  * Share ideas on how to develop the SDK further.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by viviveevee** [comment]
Template to use for Feature Requests:
``` markdown
## What problem will this feature solve?
<Explain what motivates your feature request>

## How will this feature work?
<Explain how the user would interact with this feature and how dfx would respond>

## Alternatives considered
<Are there any other ways to solve the problem? Why do they not work as well?>

#### Request source
<If this request originally came from a different place (e.g. the forum), note the origin here>
```
